### ISSUE TITLE
Missing gpu files in pypi 

### ISSUE THREAD

**Issue by Airfooox** [radis/radis#681]
### 🐛 Describe the bug

Files for the GPU acceleration functionality are missing in the pypi release archive (https://pypi.org/project/radis/#files)
compared to the archive obtained in GitHub https://github.com/radis/radis/releases/tag/0.15

Namely the missing files I found were in the radis/gpu directory: 
gpu.py, params.py, structs.py and the README. 
The cuda directory in that same gpu dir is missing a few files aswell: 
build_kernals.bat, driver.py, emulate.py

### 💡 Possible solutions

Current workaround is of course to download radis 0.15 from GitHub and implement it locally.

Im not familiar with the release procedure or how it it done generally, so I cant suggest any changes. But I think it would be good to check all files from the pypi release archive against the correct release from GitHub, maybe some other files were not included correctly.

### 🎲 Radis version

0.15

### 💻 Operating system

Windows

**Comment by minouHub** [comment]
Hello
Thank you so much for reporting this major issue!
Indeed, running `python -m build` outputs the gpu files, **but in the wheel it gives**:
```
[... a lot of stuff ...]
C:\Users\Nicolas Minesi\AppData\Local\Temp\build-env-br19d3_h\lib\site-packages\setuptools\command\build_py.py:215: _Warning: Package 'radis.gpu' is absent from the `packages` configuration.
!!

        ********************************************************************************
        ############################
        # Package would be ignored #
        ############################
        Python recognizes 'radis.gpu' as an importable package[^1],
        but it is absent from setuptools' `packages` configuration.

        This leads to an ambiguous overall configuration. If you want to distribute this
        package, please make sure that 'radis.gpu' is explicitly added
        to the `packages` configuration field.

        Alternatively, you can also rely on setuptools' discovery methods
        (for example by using `find_namespace_packages(...)`/`find_namespace:`
        instead of `find_packages(...)`/`find:`).

        You can read more about "package discovery" on setuptools documentation page:

        - https://setuptools.pypa.io/en/latest/userguide/package_discovery.html

        If you don't want 'radis.gpu' to be distributed and are
        already explicitly excluding 'radis.gpu' via
        `find_namespace_packages(...)/find_namespace` or `find_packages(...)/find`,
        you can try to use `exclude_package_data`, or `include-package-data=False` in
        combination with a more fine grained `package-data` configuration.

        You can read more about "package data files" on setuptools documentation page:

        - https://setuptools.pypa.io/en/latest/userguide/datafiles.html


        [^1]: For Python, any directory (with suitable naming) can be imported,
              even if it does not contain any `.py` files.
              On the other hand, currently there is no concept of package data
              directory, all directories are treated like packages.
        ********************************************************************************

!!
  check.warn(importable)
```
Other files are missing in the tests folder, but that is not important for users

**Comment by dcmvdbekerom** [comment]
I would propose to fix this when we merge the new GPU stuff, as the Vulkan engine will be a major rewrite

**Comment by dcmvdbekerom** [comment]
Ah I didn't realize you were that fast! Yes definitely include this fix in 0.15.1

**Comment by minouHub** [comment]
Hello,
This issue is now resolved with radis 0.15.1
Thanks for reporting
